axelor-account/src/main/resources/views/InvoiceLine.xml

```diff
@@ -428,7 +462,24 @@
     <action name="action-invoice-line-record-account-onchange"
       if="__config__.app.getApp('account').manageCutOffPeriod"/>
     <action name="action-method-account-invoice-line-account-onchange"/>
+    <action name="action-invoice-line-method-manage-invoice-line-axis"/>
+    <action name="action-method-invoice-line-axis-create-analytic-line"/>
+    <action name="action-group-invoice-line-analytic-distribution-template-onchange"/>
   </action-group>
+
+  <action-group
+    name="action-group-invoice-line-analytic-distribution-template-onchange">
+    <action name="action-method-clear-analytic-invoice-line"/>
+    <action name="action-method-check-analytic-on-invoice-line"/>
+    <action name="action-invoice-line-method-create-distribution"/>
+    <action name="action-method-print-analytic-account-on-invoice-line"/>
+  </action-group>
+
+  <action-group name="action-group-invoice-line-analytic-move-line-onchange">
+    <action name="action-method-check-analytic-account-on-invoice-line"/>
+    <action name="action-method-print-analytic-account-on-invoice-line"/>
+  </action-group>
```

Comment:
Please respect action naming convention `action-{objet name}-{action type}-{action-description}`
Same goes for new actions in the following